templates/ReactSpa/ClientApp/boot.tsx

```diff
@@ -10,7 +10,7 @@ let routes = RoutesModule.routes;
 function renderApp() {
     // This code starts up the React app when it runs in a browser. It sets up the routing
     // configuration and injects the app into a DOM element.
-    const baseUrl = document.getElementsByTagName('base')[0].getAttribute('href');
+    const baseUrl = document.getElementsByTagName('base')[0].getAttribute('href')!;
```

Comment:
When we assume that a base element exists, we can also assume that it has the `href` attribute set.